gnucash/gtkbuilder/assistant-stock-transaction.glade

```diff
@@ -0,0 +1,941 @@
+              </object>
+              <packing>
+                <property name="left-attach">1</property>
+                <property name="top-attach">2</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkBox" id="cash_amount_box">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="hexpand">True</property>
+              </object>
+              <packing>
+                <property name="left-attach">1</property>
+                <property name="top-attach">1</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkLabel" id="cash_account_label">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="halign">end</property>
+                <property name="label" translatable="yes">Cash Account</property>
+                <property name="use-underline">True</property>
+                <property name="justify">center</property>
+                <property name="mnemonic-widget">cash_memo_entry</property>
+              </object>
+              <packing>
+                <property name="left-attach">0</property>
+                <property name="top-attach">0</property>
+              </packing>
+            </child>
+            <child>
+              <placeholder/>
+            </child>
+          </object>
+          <packing>
+            <property name="expand">False</property>
+            <property name="fill">False</property>
+            <property name="position">1</property>
+          </packing>
+        </child>
+      </object>
+      <packing>
+        <property name="title" translatable="yes" comments="Dialog title for the remains of a stock split">Cash</property>
+        <property name="complete">True</property>
+      </packing>
+    </child>
+    <child>
+      <object class="GtkBox" id="fees_details_page">
+        <property name="visible">True</property>
+        <property name="can-focus">False</property>
+        <property name="border-width">6</property>
+        <property name="orientation">vertical</property>
+        <property name="spacing">12</property>
+        <child>
+          <object class="GtkLabel" id="stock_fees_title">
+            <property name="visible">True</property>
+            <property name="can-focus">False</property>
+            <property name="margin-bottom">6</property>
+            <property name="label" translatable="yes">In this page, input any broker fees incurred in this transaction. The fee may be capitalized into the asset account, or expensed into a broker fees account. Typically fees on purchase are capitalized, and fees on sales are expensed. If there are no fees involved in this transaction, it can be left blank.</property>
```

Comment:
`Typically fees on purchase are capitalized, and fees on sales are expensed.`
Doesn't this depend on the local tax law?

Reply:
It does, though mostly as a matter of timing, particularly in jurisdictions where the capital gains and ordinary income tax rates are the same. By capitalizing the commissions and fees at purchase the government defers the deduction of them until the time of sale; for sale commissions and fees it doesn't matter whether the proceeds are net of commissions and fees or deducted afterwards.

FWIW in the US one reports both the basis and proceeds as net of commissions and fees, so not the "typical" arrangement. In the US it's become largely academic: There's no commission (brokers are making their money on order flow) and there's a flat $5.00 fee from the Securities and Exchange Commission on sales.